spec/features/dams_collections_spec.rb

```diff
@@ -215,7 +215,7 @@
     scenario 'should see SHARE output' do
       sign_in_developer
       visit osf_api_dams_collection_path @provCollection.pid
-      expect(page).to have_content('"Test Title : "')
+      expect(page).to have_content('"Test Title"')
```

Comment:
could we possibly add a test that includes a translationVariant title example? that might help with coverage of the change you made, for future edits/change requests.